test/spec.js

```diff
@@ -1075,21 +1075,40 @@ module.exports = (fetchMock, theGlobal, Request, Response) => {
 						})
 				});
 
-				it('falls back to second route if first route already matched enough times', function () {
```

Comment:
I removed this test because falling-back now conflicts with the overriding behavior.  There can be only one unique matcher that exists in the `this.routes` array.

Reply:
ah, I didn't realise I explicitly supported that behaviour - the API has grown so much over the years 😵 . It makes me think that the correct way to implement this is to accept an option `{override: true}` when adding the mock

Reply:
I'm fine with that. I should be able to get to this over the holidays.